.github/workflows/mingw.yml

```
@@ -192,18 +192,93 @@ jobs:
       - name: Upload
         uses: actions/upload-artifact@v2
         with:
-          name: build-cross
+          name: build-cross-gcc-x86_64
           path: install.zip
 
-  cross-test:
-    needs: [cross]
+  cross-gcc-x86_64-test:
+    needs: [cross-gcc-x86_64]
     runs-on: windows-latest
     steps:
       - uses: actions/download-artifact@v2
         with:
-          name: build-cross
+          name: build-cross-gcc-x86_64
 
       - name: 'Run tests'
         run: |
           7z x install.zip
           ./_build/install/usr/local/bin/python3.exe -c "import sysconfig, pprint; pprint.pprint(sysconfig.get_config_vars())"
+
+
+  cross-llvm-mingw:
+    runs-on: ubuntu-18.04
```

Any reason why not use `ubuntu-latest`?
```suggestion
    runs-on: ubuntu-latest
```

---

Yes, the version of python installed by the setup-python action didn't work in the docker container, which is still based on ubuntu 18.04.  Switching the runner to 18.04 fixed that.